src/Features/LanguageServer/Protocol/ExternalAccess/Razor/FormatNewFileParams.cs

```diff
@@ -10,12 +10,12 @@ namespace Microsoft.CodeAnalysis.LanguageServer.ExternalAccess.Razor;
 [DataContract]
 internal sealed record FormatNewFileParams
 {
-    [DataMember(Name = "document")]
+    [System.Text.Json.Serialization.JsonPropertyName("document")]
     public required TextDocumentIdentifier Document { get; init; }
```

Comment:
Why not just take the file contents from protocol assemblies? This looks error prone

Reply:
> This looks error prone

Which part?

Note: we have to take the code, and do things like fixup references, names of things, namespaces, and licenses.

Reply:
Can you start with the other file contents and then automate the references, naming, and licenses automatically? Otherwise, it's a needle in a haystack to find where the newtonsoft versions were wrong before unless you're also running the protocol unit tests.